pkg/securelaunch/tpm/tpm.go

```diff
@@ -9,23 +9,18 @@ import (
 	"bytes"
 	"crypto/sha256"
 	"encoding/binary"
+	"errors"
 	"fmt"
 	"io"
 	"log"
 
-	"github.com/google/go-tpm/tpm2"
-	"github.com/google/go-tpm/tpmutil"
 	slaunch "github.com/u-root/u-root/pkg/securelaunch"
 	"github.com/u-root/u-root/pkg/securelaunch/eventlog"
+	"github.com/u-root/u-root/pkg/tss"
 )
 
-/*
- * tpm2.ReadPCR and tpm2.ExtendPCR need hashAlgo passed.
- * using sha256 for now
- */
-const (
-	hashAlgo = tpm2.AlgSHA256
-)
+var hashAlgo = tss.HashSHA256.GoTPMAlg()
```

Comment:
is hashAlgo still even used?

Reply:
if not, then GoTPMAlg can be unexported, and this just deleted.

Reply:
yes it is still used. When we marshal a pcr event(in plain words any measurement event that we want) and write to sysfs, we do send hashAlgo. This sysfs file is read/unmarshalled by tpmtool (now called txtlog) and format of this marshalling event structure if https://github.com/u-root/u-root/blob/master/pkg/txtlog/structures.go#L158
The digests parameter contains hashAlgo as one of the variables.